From c12f2431c9cb7aa65e37c86d4e7efe4230de327e Mon Sep 17 00:00:00 2001
From: Huang Chaofan <chaofan.huang@nxp.com>
Date: Wed, 11 Apr 2018 16:04:01 +0800
Subject: [PATCH 3580/5242] MLK-17999[IMX8QXP B0] Change the definition of vpu
 B0 firmware name for qxp board

commit  81857c66bd6054ca976d7be7b036ab85c5bcc702 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change the definition of vpu B0 firmware name for qxp board to follow
previous platforms

Signed-off-by: Huang Chaofan <chaofan.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu-decoder-b0/vpu_b0.h         |    2 +-
 drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mxc/vpu-decoder-b0/vpu_b0.h b/drivers/mxc/vpu-decoder-b0/vpu_b0.h
index faafcca..7c27997 100644
--- a/drivers/mxc/vpu-decoder-b0/vpu_b0.h
+++ b/drivers/mxc/vpu-decoder-b0/vpu_b0.h
@@ -45,7 +45,7 @@
 
 #define VPU_MAX_FORMATS 4
 #define VPU_MAX_BUFFER 32
-#define M0FW_FILENAME "decoder_main.bin"
+#define M0FW_FILENAME "vpu_fw_imx8qxp_dec.bin"
 #define MMAP_BUF_TYPE_SHIFT 28
 #define MMAP_BUF_TYPE_MASK 0xF0000000
 #define M0_BOOT_SIZE 0x1000000
diff --git a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h
index 532beb4..c023d75 100644
--- a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h
+++ b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.h
@@ -45,7 +45,7 @@
 
 #define VPU_MAX_FORMATS 4
 #define VPU_MAX_BUFFER 32
-#define M0FW_FILENAME "encoder_main.bin"
+#define M0FW_FILENAME "vpu_fw_imx8qxp_enc.bin"
 #define MMAP_BUF_TYPE_SHIFT 28
 #define MMAP_BUF_TYPE_MASK 0xF0000000
 #define M0_BOOT_SIZE 0x1000000
-- 
1.7.9.5

